packages/sdk/src/session/index.ts

```diff
@@ -0,0 +1,28 @@
+import { createStore } from "./../store/composed";
+
+export interface Currency {
+  code: string; // Ex: USD
+  symbol: string; // Ex: $
+}
+
+export interface Person {
+  id: string;
+  email: string;
+  givenName: string;
+  familyName: string;
```

Comment:
```suggestion
  firstName: string;
  lastName: string;
```

Reply:
I didn't really change this bit. This type was like this before. If you want, I can do another PR changing these types

Reply:
okay, seems reasonable to do it in another pr